src/services/quandl-service.js

```diff
@@ -75,9 +111,18 @@
 
     class QuandlService {
         constructor($resource) {
+            this.observable = new Observable();
             this.$resource = $resource;
         }
 
+        on() {
+            this.observable.on.apply(this.observable, arguments);
```

Comment:
Similarly to the comment above, can we circumvent `apply` here and pass the `arguments` through directly? Also, why not explicitly specify the arguments in this method's signature?